metron-deployment/packaging/docker/rpm-docker/SPECS/metron.spec

```diff
@@ -274,6 +276,23 @@ This package installs the Metron Indexing files
 %{metron_home}/config/zeppelin/metron/metron-ip-report.json
 %{metron_home}/config/zeppelin/metron/metron-connection-volume-report.json
 
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  
```

Comment:
I don't have that in my editor, do you have it in vim or something?  or is it just from what you see on the web?

line 279?